### DISCUSSION TITLE
Infinite Scroll "reversed" - still load page at top

### DISCUSSION THREAD

**Discussion by joetin-4** [quasarframework/quasar#16152] [General - Components / Directives / etc]
So my goal is to be able to have an entire q-page to display an infitite scroll in the reversed style. 
I can't however get the reversed prop to work properly. Whenever the page loads I always start at the TOP, not the bottom. What am I doing wrong here? The process is as follows:

1) I generate a clean, new quasar project

2) I clean up **IndexPage.vue** and replicate the example in the quasar-documentations (https://quasar.dev/vue-components/infinite-scroll/) as follows:

(sorry guys, also have issues displaying the code in indented style..Tried to make the best out of it)

`<template>`
`  <q-page padding class="column flex-center">`
`    <q-infinite-scroll @load="onLoad" reverse>`

`      <template v-slot:loading>`
`        <div class="row justify-center q-my-md">`
`          <q-spinner color="primary" name="dots" size="40px" />`
`        </div>`
`      </template>`

`      <div v-for="(item, index) in items" :key="index" class="caption q-py-sm">`
`        <q-badge class="shadow-1">`
`          {{ items.length - index }}`
`        </q-badge>`
`        Lorem ipsum dolor sit amet consectetur adipisicing elit. Rerum`
`        repellendus sit voluptate voluptas eveniet porro. Rerum blanditiis`
`        perferendis totam, ea at omnis vel numquam exercitationem aut, natus`
`        minima, porro labore.`
`      </div>`

`    </q-infinite-scroll>`
`  </q-page>`
`</template>`

(And then in the script-tag i have items, onLoad() just like in the documentation)

I make no relevant changes to **/layouts/MainLayout.vue** or **App.vue**. In fact, I don't make any changes anywhere except **IndexPage.vue**.
Why doesnt this work? Please help me understand.

**Comment by sergey-yarchuk-4ib** [comment]
I have the same problem. Solved by wrapping in
```vue
<q-scroll-area> ... <q-scroll-area />
```

```vue
<q-scroll-area ref="ScrollAreaRef" style="height: 200px;">
	<q-infinite-scroll reverse @load="onLoad">
		<div v-for="(item, index) in items" :key="index" class="caption q-py-sm">
			<q-badge class="shadow-1">
				{{ items.length - index }}
			</q-badge>
			Lorem ipsum dolor sit amet consectetur adipisicing elit. Rerum repellendus sit voluptate voluptas eveniet porro. Rerum blanditiis perferendis totam, ea at omnis vel numquam exercitationem aut, natus minima, porro labore.
		</div>

		<template #loading>
			<div class="row justify-center q-my-md">
				<q-spinner color="primary" name="dots" size="40px" />
			</div>
		</template>
	</q-infinite-scroll>
</q-scroll-area>
```
